src/components/new-post.vue

```diff
@@ -1,25 +1,40 @@
 <template>
+  <ion-header>
+   <ion-toolbar>
+    <ion-buttons slot="start" class="back-button">
+        <ion-back-button @click="closeModal"/>
+      </ion-buttons>
+      <ion-title>
+       Create a Post
+      </ion-title>
+      <ion-button slot="end" @click="submit" data-cy="submitPost" type="submit" data-cy-role="submit" fill="outline" v-bind:disabled="!canSubmit"
+       shape="round" size="small">
+        <ion-icon :icon="sendIcon"></ion-icon>
+      </ion-button> 
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
```

Comment:
As outlined later, the same widget should be visible outside of a modal in regular desktop mode. As such I recommend doing a wrapper component instead of editing this component (similar as `login-modal.vue` is a warpper to `login.vue` and allows the inner component to still be used directly).